Container installation jcayouette (#2745)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/deploy-as-a-container.adoc

= Deploying {productname} as a container
// remove this attribute at publishing time
:uyuni-content: true

//The process of installing {productname} server was changed in version {productnumber}. 
//{productname} is now deployed as a container.

ifeval::[{uyuni-content} == true]
== Deployment preparations

> In this section, you will gain expertise in setting up and deploying a {productname} Server. The process encompasses the installation of [command]``Podman``, [filename]``Uyuni container utilities``, deployment, and then initiating your interaction with the container through [command]``mgrctl``.

Reviewer: ```suggestion
In this section, you will gain expertise in setting up and deploying a {productname} Server.
The process encompasses the installation of [command]``Podman``, [systemitem]``Uyuni container utilities``, deployment, and then initiating interaction with the container through [command]``mgrctl``.
```

> It is important to note that this section assumes you have already configured a {tumbleweed} or {opensuse} {opensuse-version} server, whether it's running on a physical machine or within a virtual environment.

Reviewer: ```suggestion
[NOTE]
====
This section assumes you have already configured a {tumbleweed} or {opensuse} {opensuse-version} server, whether it is running on a physical machine or within a virtual environment.
====
```

== Installing Podman on openSUSE Leap {opensuse-version} and {tumbleweed}

To install [command]``Podman`` on either {opensuse} {opensuse-version} or {opensuse} {tumbleweed}, proceed with the following steps.

.Procedure: Installing Podman
. Open a terminal window.

. Update the package manager's repository information to ensure you have the latest package information:
+

[source,shell]
----
sudo zypper refresh
----

. Install [command]``Podman`` by running the following command:
+

[source,shell]
----
sudo zypper -y install podman
----

> 
> . Zypper will download and install [command]``Podman`` along with its dependencies.

Reviewer: ```suggestion
+

Zypper will download and install [command]``Podman`` along with its dependencies.
```
This is not a separate step, but result of the previous one ;)

> . Once the installation is complete, verify that [command]``Podman`` is installed by running:

Reviewer: ```suggestion
. When the installation is complete, verify that [command]``Podman`` is installed by running:
```

+

[source,shell]
----
podman --version
----
+

This command should display the installed version of [command]``Podman``.
You should now have [command]``Podman`` successfully installed on your openSUSE Leap 15.5 system, and you can start using it to manage containers on your system.

> For more information see, link:https://podman.io/[https://podman.io]

Reviewer: ```suggestion
For more information, see https://podman.io/.
```
